public/stylesheets/sass/_landing.scss

```diff
@@ -0,0 +1,74 @@
+.landing-hero {
+  background-image: url('../img/tricity-panorama.svg');
+  background-size: contain;
+  background-repeat: no-repeat;
+  background-position: bottom center;
+  height: 85vh;
+}
+
+.landing-community {
+  background-image: url('../img/bg-community.png');
+  height: 265px;
+}
+
+.landing-bg-rubies {
+  background-image: url('../img/rubies.svg');
+  background-repeat: no-repeat;
+  background-position-x: right;
+}
+
+.landing-link {
+  font-size: 1.1rem;
+}
+
+.landing-map-canvas {
+  width: 500px;
```

Comment:
@aokon I've changed the map dimensions by setting the width to 100% (it fits the column container then, so it's responsive) and setting the height to the fixed number of pixels. The `vh` didn't work well in these case - the map had the different proportions on each screen.